spec/features/user_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :system do
+  subject { User.new(name: 'John', posts_counter: 30, photo: 'https://randomuser.me/api/portraits/men/70.jpg', bio: 'Teacher from Poland.') }
+
+  before { subject.save }
+
+  describe 'index page' do
+    it 'I can see the username of all other users.' do
+      visit root_path(subject)
+      page.has_content?(subject.name)
+    end
+
+    it 'I can see the profile picture for each user.' do
+      visit root_path(subject)
+      page.has_css?('.img-fluid')
+    end
+
+    it 'I can see the number of posts each user has written.' do
+      visit root_path(subject)
+      page.has_content?(subject.posts_counter)
+    end
+
+    it "When I click on a user, I am redirected to that user's show page." do
```

Comment:
- [ ] Please kindly make sure all your tests are passing. Actually, one test doesn’t pass. The test redirects to the user's show page when you click on a user fail kindly fix that 🙃 😉

<img width="1163" alt="Screen Shot 2022-12-05 at 16 24 03" src="https://user-images.githubusercontent.com/74501523/205649469-6eb98063-3bf7-4585-a3c4-fa066e7c6920.png">